src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export default function ArticleCard({ article }) {
```

Comment:
If you wanted, you could try to destructure this further to stop you having to write 'article.' in order to access each key. (It works okay as it is, but this would make it even better)